packages/ui/src/components/PreviewItem/PreviewItem.tsx

```diff
@@ -0,0 +1,66 @@
+import MagnifyingGlassPlusIcon from "@heroicons/react/24/outline/MagnifyingGlassPlusIcon";
+import { ReactNode } from "react";
+import { CardBasic } from "../Card";
+import Button from "../Button";
+
+export interface PreviewItemProps {
+  title: string;
+  details: ReactNode[];
+  buttonName: string;
+}
+
+const PreviewItem = ({ title, details, buttonName }: PreviewItemProps) => {
+  return (
+    <div
+      data-h2-position="base(relative)"
+      data-h2-display="base(grid)"
+      data-h2-grid-template-columns="base(3fr 1fr)"
```

Comment:
Using a 1/3 grid for the icon results in quite a lot of wasted whitespace.  Maybe a horizontal flexbox where the icon uses just the right amount of whitespace and the left side can grow as needed could work better?
![image](https://github.com/user-attachments/assets/b77eb2ba-771c-47b9-a761-f994b86c345d)

Compare to Figma:
![image](https://github.com/user-attachments/assets/0ab8fafa-9ad2-4dce-9a3d-41075dd3f136)